src/main/java/nl/tudelft/contextproject/model/entities/Health.java

```diff
@@ -0,0 +1,27 @@
+package nl.tudelft.contextproject.model.entities;
+
+/**
+ * Interface for entities with health.
+ */
+public interface Health {
+	/**
+	 * Set the health of the object.
+	 * 
+	 * @param newHealth
+	 * 		the new health
+	 */
+	public void setHealth(float newHealth);
+	/**
+	 * Make the object take damage.
+	 * 
+	 * @param damage
+	 * 		the damage dealt to the object
+	 */
+	public void takeDamage(float damage);
+	
+	/**
+	 * @return
+	 * 		the health of this objects
+	 */
+	public float getHealth();
```

Comment:
Remove "public" in the interface, to conform with the style of the other interfaces.